web_src/js/components/ContextPopup.vue

```diff
@@ -91,16 +91,21 @@ export default {
 <template>
   <div ref="root">
     <div v-if="loading" class="tw-h-12 tw-w-12 is-loading"/>
-    <div v-if="!loading && issue !== null">
-      <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
-      <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
-      <p>{{ body }}</p>
+    <div v-if="!loading && issue !== null" class="tw-flex tw-flex-col tw-gap-2">
+      <div class="tw-text-12">{{ issue.repository.full_name }} on {{ createdAt }}</div>
+      <div><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</div>
+      <div v-if="body">{{ body }}</div>
       <!-- eslint-disable-next-line vue/no-v-html -->
-      <div v-html="renderedLabels"/>
+      <div class="labels-list-wrapper" v-html="renderedLabels"/>
     </div>
     <div v-if="!loading && issue === null">
-      <p><small>{{ i18nErrorOccurred }}</small></p>
-      <p>{{ i18nErrorMessage }}</p>
+      <div class="tw-text-12">{{ i18nErrorOccurred }}</div>
+      <div>{{ i18nErrorMessage }}</div>
     </div>
   </div>
 </template>
+<style scoped>
+.labels-list-wrapper:has(.labels-list:empty) {
```

Comment:
Why not just as simple as `v-if=renderedLabels`

Reply:
If  backend would return a empty string, it would work with `v-if` but then we need to modify more places I assume.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/30824/commits/87cc2ed06ced8bf6a8de41354f17e4f48b30e0d7.